test/unit/news_view_model.dart

```diff
@@ -105,10 +110,41 @@ class AlreadyExistingNewsVM extends NewsViewModel {
   }
 }
 
+class MockOfflineRecorder implements OfflineRecorder {
+  @override
+  Future<void> downloadNews(News news) async {
+    //does not need to do anything
+  }
+
+  @override
+  int getCurrentMaxStorageSize() {
+    // TODO: implement getCurrentMaxStorageSize
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<News> loadNews(String date) async {
+    // TODO: implement loadNews
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<int> setMaxStorageSize(int newStorageSize) async {
+    // TODO: implement setMaxStorageSize
+    throw UnimplementedError();
+  }
+}
```

Comment:
```suggestion
class MockOfflineRecorder extends Fake implements OfflineRecorder {
  @override
  Future<void> downloadNews(News news) async {
    //does not need to do anything
  }
}
```

Using `extends Fake` will automatically replace all unimplemented methods with a stub throwing `UnimplementedError`.